src/modules/organisation/persistence/organisation.repository.ts

```diff
@@ -341,6 +348,7 @@ export class OrganisationRepository {
             .select('*')
             .where(whereClause)
             .offset(searchOptions.offset)
+            .orderBy([{ kennung: QueryOrder.ASC_NULLS_FIRST }, { name: QueryOrder.ASC_NULLS_FIRST }])
```

Comment:
Einträge mit Null z.B. Orgas vom Typ "LAND" sind trotzdem nicht oben. Funktioniert hier das nicht richtig mit NULLS_FIRST?

Reply:
In der Response des Servers sind das die ersten. Das scheint im FE umsortiert zu werden. Da das aber nicht im Scope des Tickets ist, wollte ich dafür keinen extra FE-Branch aufmachen.

Reply:
Alles klar, Ich kümmere mich darum